docs/data/material/components/multiple-snackbars/multiple-snackbars.md

```diff
@@ -0,0 +1,89 @@
+---
+product: material-ui
+title: Multiple Snackbars React component
+components: SnackbarsProvider
+materialDesign: https://material.io/components/snackbars
+waiAria: https://www.w3.org/TR/wai-aria-1.1/#alert
+---
+
+# Multiple Snackbars
```

Comment:
@samuelsycamore could you please review the content of this page?

Reply:
Also, what should we do with:
1. https://mui.com/material-ui/react-snackbar/#consecutive-snackbars
2. https://deploy-preview-31991--material-ui.netlify.app/material-ui/react-snackbar/#notistack

Reply:
I've done a bit of writing revision there but would love Sam's input as well regardless!

I've got a bit confused by the name in relation to what's being introduced. As far as I understood, there's a new `SnackbarsProvider` component and a `useSnackbars` hook. First question would be: wouldn't make more sense for the hook to be in Base? Second is: instead of placing the `SnackbarsProvider` in the lab, shouldn't we add it as a supporting component for the Snackbar?

That would allow having all the content inside the Snackbar, which seems to make a lot more sense to me. We'd avoid content and demo repetition while also connecting the narrative that notistack used to be what enabled multiple snackbars but now we have a "native" solution for that, "deprecating" the need to depend on notistack for that.

Reply:
`useSnackbars` is not a low level hook for unstyled Snackbar to be it in Base. It's a hook available for the applications where it can be used within their components to trigger a snackbar inside a `SnackbarsProvider` context.

While I agree that I will be easier to manage if it is moved within core's Snackbar for all the reasons you mentioned, it is supposed to be in lab since this is first version and we can would get more feedback from users regarding bugs, design, API design which is what lab is used for initially. 
Once it's robust, we move it to core which will make things much easier then (regarding repetitive demos etc.)

For now we can still show there is a replacement of notistack available in lab by having a link to this page. I have also added a link to multiple snackbars from Core's snackbar [here](https://deploy-preview-31991--material-ui.netlify.app/material-ui/react-snackbar/#main-content) in the `Frequency` section.  

What do you think?

Reply:
Right, that makes sense. I changed the word slightly from _multiple_ to _stacked_ snackbars, which sounds clearer to me. Additionally, edited the documentation to showcase that notistack and now this component are both alternatives to stack snackbars, trying to make it clear that the latter is our native take on it.